package.json

```diff
@@ -15,6 +16,12 @@
     "node",
     "daemon"
   ],
+  "standard": {
+    "ignore": [
+      "/test/*",
```

Comment:
Because mocha does this magic of adding vars without having them been declared

```
standard: Use JavaScript Standard Style (https://github.com/feross/standard)
  /Users/david/Documents/code/ipfs/node-ipfsd-ctl/test/test.js:8:1: "describe" is not defined.
  /Users/david/Documents/code/ipfs/node-ipfsd-ctl/test/test.js:11:3: "before" is not defined.
  /Users/david/Documents/code/ipfs/node-ipfsd-ctl/test/test.js:22:3: "it" is not defined.
  /Users/david/Documents/code/ipfs/node-ipfsd-ctl/test/test.js:29:3: "before" is not defined.
  /Users/david/Documents/code/ipfs/node-ipfsd-ctl/test/test.js:47:3: "it" is not defined.
  /Users/david/Documents/code/ipfs/node-ipfsd-ctl/test/test.js:50:3: "it" is not defined.
  /Users/david/Documents/code/ipfs/node-ipfsd-ctl/test/test.js:55:1: "describe" is not defined.
  /Users/david/Documents/code/ipfs/node-ipfsd-
...
```

but got it, I can add a comment to solve this, fixing now :)